nx/include/net/if.h

```diff
@@ -0,0 +1,547 @@
+// Removed kernel stuff, use anon unions
```

Comment:
Is this actually required by any implemented API?

Reply:
Yes, by all network interface ioctls, for example it's needed to get the IP of the console (if I get around the 0xce01 issue).